src/test/test.spec.ts

```diff
@@ -0,0 +1,22 @@
+import * as request from 'supertest';
+import app from '@api';
+
+describe('GraphQL - UserResolver - Create', () => {
+  before(async () => {
+    await app.run();
+  });
+
+  const requestMaker = request(`http://localhost:5000`).post('/graphql').set('Accept', 'application/json');
+
+  it('should return -1 when the value is not present', async () => {
+    await requestMaker
+      .send({
+        query: '{ hello }',
+      })
+      .expect('Content-Type', /json/)
+      .expect(200)
+      .expect((res) => {
+        console.log(res.body.data.hello);
+      });
```

Comment:
Até onde eu tinha trabalhado, tipagem do supertest não tá clara o suficiente, mas senão me engano ele retorna Promise (tanto q vc tá usando await). Então não precisa do callback:

```suggestion
  const requestMaker = request(`http://localhost:5000`).post('/graphql').set('Accept', 'application/json');

  it('should return -1 when the value is not present', async () => {
    const res = await requestMaker
      .send({
        query: '{ hello }',
      })
      .expect('Content-Type', /json/)
      .expect(200);

      console.log(res.body.data.hello);
```

Reply:
Pra usar callbacks corretamente vc deveria usar o `done` do Mocha tbm, aí começa a ficar mais chato.

Reply:
E esse log aí não precisa né?